contrib/mod_tls.c

```diff
@@ -10674,13 +10677,28 @@ MODRET tls_prot(cmd_rec *cmd) {
     return PR_ERROR(cmd);
   }
 
+  c = find_config(main_server->conf, CONF_PARAM, "TLSOptions", FALSE);
+  while (c != NULL) {
+    unsigned long opts = 0;
+
+    pr_signals_handle();
+    opts = *((unsigned long *) c->argv[0]);
+    tls_opts |= opts;
+
+    c = find_config_next(c, c->next, CONF_PARAM, "TLSOptions", FALSE);
+  }
```

Comment:
You probably don't need this `while` loop here; the `TLSOptions` configuration is looked up at session initialization time, and stored in the `tls_opts` file-global variable, which means that the above loop is probably redundant.

Unless you found it was needed for some other reason?